docs/documentation/publishing-on-heroku.md

```diff
@@ -43,22 +43,20 @@ You'll need to have [put your code on GitHub](/docs/github-desktop) to use this
 
 Your prototype will deploy automatically each time you push your code to GitHub (it takes a few minutes each time).
 
-## Set a username and password
+## Set a password
 
-We need to set a username and password or the Prototype Kit won’t run online. They don’t have to be complicated – it’s just to stop people accidentally coming across your prototype online and mistaking it for a real service.
+You need to set a password or the Prototype Kit will not run online. This password does not have to be complicated. It's just to stop people accidentally finding your prototype online and mistaking it for a real service.
 
-1. At the top click the **Settings** tab.
+1. At the top of the Heroku UI, click the **Settings** tab.
 
 2. Click **Reveal config vars**.
 
-3. In KEY put the word USERNAME
+3. In KEY, enter the word PASSWORD. Do not include the full stop.
 
-4. In VALUE put a username of your choice, click **Add**.
+4. In VALUE, enter a password of your choice and click **Add**.
 
-That will be saved and you can add another KEY and VALUE.
+5. In the top right of the Heroku UI, click **Open app** to see your prototype online.
 
-5. In KEY put the word PASSWORD
+### If you get an error about username
 
-6. In VALUE put a password of your choice, click **Add**.
-
-7. In the top right, click **Open app** to see your prototype online!
+Repeat the steps for setting a password.
```

Comment:
@joelanman I'm pretty sure this is wrong...I added it because I looked at the screenshots from our chat, and it seemed like the steps for here were identical to the ones in the 'Set a password' process above. Anyway, advice welcome! :)

Reply:
D'oh, yes - my bad. Thanks, will fix.

Reply:
Thanks!